ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java

```diff
@@ -94,6 +97,13 @@ public MarkSweepPruner(
             BesuMetricCategory.PRUNER,
             "sweep_operations_total",
             "Total number of sweep operations performed");
+
+    markStopwatch = Stopwatch.createUnstarted();
+    metricsSystem.createLongGauge(
```

Comment:
Is MarkSweepPruner a singleton for the life of the java process?  Counters can handle multiple counters, gauges will throw an exception when the second gauge is created.

Reply:
Yup, it's a singleton